openblas-src/build.rs

```diff
@@ -104,6 +104,10 @@ fn build() {
     } else {
         cfg.no_static = true;
     }
+    cfg.target = match env::var("OPENBLAS_TARGET") {
+        Ok(target) => target.parse().ok(),
```

Comment:
Given that the error is ignored, I propose to remove `ParseTargetError` all together. Just use some existing struct for satisfy the trait.

Reply:
`ParseTargetError` is merged in `openblas_build::error::Error::UnsupportedTarget` cee2fd07c0edc73735d0a23d935d29777dc55dd2